content/docs/service-level-objectives.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Service Level Objectives
+date: 2022-03-31
+weight: 10
+---
+
+# Service Level Objectives
+
+Packit is using the concept of [Service level
+objectives](https://sre.google/sre-book/service-level-objectives/) defined in
+Google's SRE book. It means that when we run out of error budget, we stop all
+the feature work and work on stabilizing the service to get our budget back.
+
+We have 3 objectives defined for Packit right now.
+
+Screenshots below are not being updated in real time. They are just a snapshot
+of the state in time when this document was written so that you can see some
+actual numbers.
```

Comment:
```suggestion
*Screenshots below are not being updated in real time. They are just a snapshot
of the state in time when this document was written so that you can see some
actual numbers.*
```